google/cloud/speech/v2/samples/speech_client_samples.cc

```diff
@@ -0,0 +1,90 @@
+// Copyright 2022 Google LLC
```

Comment:
Normally the scaffold would generate a CMakeLists.txt and BUILD file in this directory.  You need to manually create them:

https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/speech/samples/BUILD.bazel

Reply:
I opted to build the samples from the library's top level BUILD and CMakeLists.txt. The top level build files will need to know where the files are anyway. And we don't need to play the `add_subdirectory` game.